src/components/ProgressCircle.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import React from "react";
+import { Svg, Circle, G, Text } from "react-native-svg";
+import colors from "../colors";
+
+type Props = {
+  // float number between 0 and 1
+  progress: number,
+  size?: number,
+  progressColor?: string,
+  backgroundColor?: string,
+};
+
+const R = 26;
+const Center = 38;
```

Comment:
purely code style but

```suggestion
const CENTER = 38;
```